### PR TITLE
adds options for -f -F -r and -e

### PR DESCRIPTION
**Pull request by bwlang**
re #1524 

I'd like to get some initial feedback on this before I do more on it... thanks!

### CODE REVIEW

**Comment by bgruening** [comment]
@bwlang looks good for me. You need to increase the tool version number and maybe add/extend a test.

**Comment by bwlang** [comment]
I think this is ready for review.

File: tools/bedtools/coverageBed.xml [hunk 1]
```diff
@@ -14,6 +14,14 @@
         $hist
         $split
         $strandedness
+        #if str($overlap_a) != "None" and str($overlap_a):
+          -f $overlap_a
```
Comment (bgruening) [inline tools/bedtools/coverageBed.xml]: these are text parameters and should be single quoted
I'm wondering if we should also add some sanitize to the text field.

**Comment by bwlang** [comment]
@bgruening  any further changes needed for this PR?

**Comment by bgruening** [comment]
Yes, looks great. Thanks.

**Pull request merged**